docs/content/usage/use-cases/credential-leak.md

```diff
@@ -8,30 +8,29 @@ weight: 5
 
 ## Overview 
 
-Numerous attack techniques(from phishing to dumpster diving) might result in a credential leak, 
+Numerous attack techniques (from phishing to dumpster diving) might result in a credential leak, 
 which can be **extremely costly** as demonstrated in our report [IResponse to IEncrypt](https://www.guardicore.com/2019/04/iresponse-to-iencrypt/).
 
-Infection Monkey can help assess the impact of stolen credentials by automatically searching 
-where these credentials can be reused.
+The Infection Monkey can help you assess the impact of stolen credentials by automatically searching 
+where bad actors can reuse these credentials in your network.
 
 ## Configuration
 
-- **Exploits -> Credentials** After setting up the Island add the users' **real** credentials 
-(usernames and passwords) to the Monkey's configuration (Don't worry, this sensitive data is not accessible and is not
- distributed or used in any way other than being sent to the monkeys, and can be easily eliminated by resetting the Monkey Island's configuration).
-- **Internal -> Exploits -> SSH keypair list** Monkey automatically gathers SSH keys on the current system. 
-For this to work, Monkey Island or initial Monkey needs to have access to SSH key files(grant permission or run Monkey as root).
-To make sure SSH keys were gathered successfully, refresh the page and check this configuration value after you run the Monkey
+- **Exploits -> Credentials** After setting up the Monkey Island, add your users' **real** credentials 
+(usernames and passwords) here. Don't worry; this sensitive data is not accessible, distributed or used in any way other than being sent to the Infection Monkey agents. You can easily eliminate it by resetting the configuration of your Monkey Island.
+- **Internal -> Exploits -> SSH keypair list**  When enabled, the Infection Monkey automatically gathers SSH keys on the current system. 
+For this to work, the Monkey Island or initial agent needs to access SSH key files (grant permission or run The Infection Monkey as root).
```

Comment:
We shouldn't recommend that Monkey Island is run as root or encourage people to change the permissions of their ssh keys.